cloud/scope/powervs_machine_test.go

```diff
@@ -99,95 +110,1003 @@ func setupPowerVSMachineScope(clusterName string, machineName string, imageID *s
 	}
 }
 
-func newPowerVSInstance(name, networkID, mac string) *models.PVMInstance {
-	return &models.PVMInstance{
-		ServerName: ptr.To(name),
-		Networks: []*models.PVMInstanceNetwork{
-			{
-				NetworkID:  networkID,
-				MacAddress: mac,
+func newPowerVSInstance(name, networkID, mac string) *models.PVMInstance {
```

Comment:
if you think its feasible, Move these helper constructor to common_test.go.

Reply:
This initialisation isn't getting used anywhere other this file - powervs_machine_test.go